dorpsgek_github/__main__.py

```diff
@@ -1,43 +1,59 @@
+import asyncio
+import importlib
 import logging
-import http
 
 from aiohttp import web
 
-from dorpsgek_github import github
-from dorpsgek_github.config import GITHUB_APP_PORT
-
-# List of modules that are enabled
-from dorpsgek_github.modules import (  # noqa
-    ping,
-    registration,
+from dorpsgek_github import config
+from dorpsgek_github.core.helpers.aiohttp_web import (
+    prepare_app,
+    run_apps,
 )
-
+from dorpsgek_github.core.aiohttp_apps.github import (
+    github_handler,
+    github_startup,
+)
+from dorpsgek_github.core.aiohttp_apps.runner import runner_handler
+from dorpsgek_github.core.load_config import load_config
+from dorpsgek_github.core.scheduler.scheduler import schedule_runner
 
 log = logging.getLogger(__name__)
+MODULES_ALWAYS_LOAD = ["ping", "registration"]
+KEYWORDS_ALWAYS_LOAD = ["dorpsgek", "environment", "only", "when"]
 
 
-async def github_handler(request):
-    headers = request.headers
-    data = await request.read()
+def create_web_apps():
+    apps = []
 
-    try:
-        await github.process_request(headers, data)
-        return web.Response(status=http.HTTPStatus.OK)
-    except Exception:
-        log.exception("Failed to handle GitHub event!")
-        return web.Response(status=http.HTTPStatus.INTERNAL_SERVER_ERROR)
+    github_app = web.Application()
+    github_app.router.add_post("/", github_handler)
+    github_app.on_startup.append(github_startup)
+    apps.append(prepare_app(github_app, port=config.GITHUB_APP_PORT))
 
+    runner_app = web.Application()
+    runner_app.router.add_get("/", runner_handler)
+    apps.append(prepare_app(runner_app, port=config.RUNNER_PORT))
 
-async def github_startup(app):
-    await github.startup()
+    return apps
 
 
 def main():
-    app = web.Application()
-    app.router.add_post("/", github_handler)
-    app.on_startup.append(github_startup)
+    logging.basicConfig(level=logging.INFO)
+    load_config()
+
+    for module in config.MODULES.split() + MODULES_ALWAYS_LOAD:
```

Comment:
Have you thought about module load order at all? I'd expect the "always load" modules to be loaded before any config modules